addons/account/models/account_move.py

```diff
@@ -3319,9 +3319,8 @@ def create(self, vals_list):
                 tax = repartition_line.invoice_tax_id or repartition_line.refund_tax_id
                 vals['tax_exigible'] = tax.tax_exigibility == 'on_invoice'
             elif vals.get('tax_ids'):
-                tax_ids = [v['id'] for v in self.resolve_2many_commands('tax_ids', vals['tax_ids'], fields=['id'])]
-                taxes = self.env['account.tax'].browse(tax_ids).flatten_taxes_hierarchy()
-                vals['tax_exigible'] = not any(tax.tax_exigibility == 'on_payment' for tax in taxes)
+                taxes = self.new({'tax_ids': vals['tax_ids']}).tax_ids.flatten_taxes_hierarchy()
+                vals['tax_exigible'] = len(taxes.filtered(lambda t: t.tax_exigibility == 'on_payment')) < 1
```

Comment:
Why did you change this?  It was working fine, IMO.
```suggestion
                vals['tax_exigible'] = not any(tax.tax_exigibility == 'on_payment' for tax in taxes)
```

Reply:
I think this was the product of a bad merge conflict resolution